zingolib/src/lightclient/deprecated.rs

```diff
@@ -72,13 +72,13 @@ impl LightClient {
             .sapling_notes
             .iter()
             .filter(|nd| nd.is_change)
-            .map(|nd| nd.note.value().inner())
+            .map(|nd| nd.sapling_note_crypto.value().inner())
```

Comment:
this is a note from the sapling_crypto crate so we should call all these notes `saping_crypto_note` instead of `sapling_note_crypto`